src/containers/InboxPage/InboxPage.stateDataRescueBooking.js

```diff
@@ -0,0 +1,53 @@
+import {
+  TX_TRANSITION_ACTOR_CUSTOMER as CUSTOMER,
+  TX_TRANSITION_ACTOR_PROVIDER as PROVIDER,
+  CONDITIONAL_RESOLVER_WILDCARD,
+  ConditionalResolver,
+} from '../../transactions/transaction';
+
+// Get UI data mapped to specific transaction state & role
+export const getStateDataForRescueBookingProcess = (txInfo, processInfo) => {
+  const { transactionRole } = txInfo;
+  const { processName, processState, states } = processInfo;
+  const _ = CONDITIONAL_RESOLVER_WILDCARD;
+
+  return new ConditionalResolver([processState, transactionRole])
+    .cond([states.INQUIRY, _], () => {
```

Comment:
CONDITIONAL_RESOLVER_WILDCARD is a variable declared in file transaction.js, with the value of '*'. In the ConditionalResolver class in file transaction.js, it is used to check whether the items in the conditions parameter are wildcards or not. If it is wildcard, the loop in the reduce method will return true, and in the end it will assign the value to isMatch variable. If the isMatch is true, then this.resolver will be assigned to the resolver parameter and then will be called at the end of the method chain.